backend/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "jsmp-backend",
+  "version": "1.0.0",
+  "description": "javascript mentoring program 2020",
+  "main": "index.js",
```

Comment:
Add the next field: `"type": "module",` here. It will allow you to use `import` instead of `require` in your `.js/.ts` files.